src/binaryen-c.h

```diff
@@ -103,6 +103,7 @@ typedef void* BinaryenFunctionTypeRef;
 // Add a new function type. This is thread-safe.
 // Note: name can be NULL, in which case we auto-generate a name
 BinaryenFunctionTypeRef BinaryenAddFunctionType(BinaryenModuleRef module, const char* name, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams);
+BinaryenFunctionTypeRef BinaryenGetFunctionType(BinaryenModuleRef module, BinaryenType result, BinaryenType* paramTypes, BinaryenIndex numParams);
```

Comment:
My use case here is that I am compiling a couple of functions (i.e. malloc etc.) from C first, then loading this into binaryen.js for further processing (adding more functions). Without `BinaryenGetStructuralFunctionType` (or how it'll be named finally), there is no way to obtain a reference to any of the existing signatures, and I had to create a duplicate signature instead.